src/ObjectUtils/CoffFile.cpp

```diff
@@ -366,6 +369,94 @@ bool CoffFileImpl::HasDebugSymbols() const {
   return false;
 }
 
+ErrorMessageOr<orbit_grpc_protos::ModuleSymbols>
+orbit_object_utils::CoffFileImpl::LoadSymbolsFromExportTable() {
+  if (!HasExportTable()) {
+    return ErrorMessage("PE/COFF file does not have an Export Table.");
+  }
+
+  ErrorMessageOr<std::vector<UnwindRange>> unwind_ranges_or_error = GetUnwindRanges();
+  if (!unwind_ranges_or_error.has_value()) {
+    return ErrorMessage{
+        absl::StrFormat("Unable to assign sizes to symbols from the Export Table: %s",
+                        unwind_ranges_or_error.error().message())};
+  }
+  absl::flat_hash_map<uint64_t, uint64_t> unwind_range_start_to_size;
+  for (const UnwindRange& unwind_range : unwind_ranges_or_error.value()) {
+    unwind_range_start_to_size.emplace(unwind_range.start, unwind_range.end - unwind_range.start);
+  }
+
+  ModuleSymbols module_symbols;
+  for (const llvm::object::ExportDirectoryEntryRef& ref : object_file_->export_directories()) {
+    // From https://docs.microsoft.com/en-us/windows/win32/debug/pe-format#export-address-table:
+    // "A forwarder RVA exports a definition from some other image, making it appear as if it were
+    // being exported by the current image." So these symbols don't really belong to this file, and
+    // we skip them.
+    bool is_forwarder{};
+    if (llvm::Error error = ref.isForwarder(is_forwarder)) {
```

Comment:
It actually confused me quite a bit, that `isForwarder` was taking a reference to write the output. Maybe add a super brief comment?

Reply:
Ok, I added it here. But note that this is the interface of the LLVM method and we can't really add such a comment everywhere this happens.